.circleci/config.yml

```diff
@@ -64,16 +67,22 @@ workflows:
                   ignore: main
 
   publish:
+    when: << pipeline.git.tag >>
```

Comment:
this workflow will only run when a new tag is created; this tag corresponds to the version of the orb